extra_geom/detectors.py

```diff
@@ -617,6 +618,7 @@ class AGIPD_1MGeometry(DetectorGeometryBase):
     You won't normally instantiate this class directly:
     use one of the constructor class methods to create or load a geometry.
     """
+    detector_type_name = 'AGIPD-1M'
```

Comment:
You use `-` as separator here, but `_` for other classes. Is it intended?

Reply:
I think it should be `-` for all, as this is going into a human-readable comment.

Reply:
Allright, "-" in the detector name strings, and "_" for "-" in the class names, where required.